modules/tensor_mechanics/examples/bridge/bridge_large_strain.i

```diff
@@ -68,11 +64,14 @@
 []
 
 [BCs]
-  [./PressureTM]
+  [./Pressure]
     [./load]
       #Applies the pressure
       boundary = top
       factor = 5e5 # Pa
+      disp_x = disp_x
```

Comment:
Yeah, I saw that as well. I'll put in an issue and have one of my students change it. It would be good practice.